src/omni-slider.js

```diff
@@ -1,532 +1,540 @@
-/* Constructor function
- * elementContainer - this acts as a wrapper for the slider, all of the sliders DOM elements will be transcluded inside the <element> provided
- * options - contains the options for the slider
- */
-function Slider(elementContainer, options) {
+(function() {
```

Comment:
this is actually being done by a gulp process, `gulp.iife()`, if you remove that from the build process this shoulw be fine.